### PR TITLE
#159236203 Relate WorkoutSession & WorkoutLog Models.

### PR DESCRIPTION
**Pull request by RrNn**
#### What does this PR do?
- This PR creates a foreign key `session_id` on the WorkoutLog model referencing the WorkoutSession 
### How should this be manually tested?
-  Clone the application as guided in the README.
- Create virtual environment and activate it.
- `pip install -r requirements_devel.txt`
- `pip install psycopg2_binary`
- `-invoke create-settings --settings-path ./settings.py`
- `npm install bower`
- `sudo python manage.py bower_install --allow-root`
- `invoke bootstrap-wger --settings-path ./settings.py --no-start-server`
- `git checkout ch-wrkoutlog-sessn-rlation-159236203`
- Make and run the migrations by:
         - `python manage.py makemigrations`
         - `pyothn manage.py migrate`
- Login the application as any user and add a workout  at `http://127.0.0.1:8000/en/workout/overview`
- Add a weight log to that day and check out the database tables `manager_workoutlog` and `manager_workoutsession`. Notice the primary key on the `manager_workoutsession` table being referenced by the `session_id` column on the `manager_workoutlog` table.
#### Relevant Pivotal Tracker story.
[#159236203](https://www.pivotaltracker.com/story/show/159236203)

### CODE REVIEW

File: wger/manager/models.py [hunk 1]
```diff
@@ -725,6 +725,7 @@ class WorkoutLog(models.Model):
     '''
 
     date = Html5DateField(verbose_name=_('Date'))
+    session_id = models.ForeignKey('WorkoutSession', null=True, blank=True, to_field='id')
```
Comment (solnsubuga) [inline wger/manager/models.py]: @RrNn  you cannot name this `session_id` because in this case, it's an instance of a WorkoutSession.

**Review by solnsubuga — CHANGES_REQUESTED** [review]
Please change the name of instance of WorkoutSession from `session_id` to `session`.

**Comment by RrNn** [comment]
@solnsubuga done.

**Comment by solnsubuga** [comment]
@RrNn  I don't see the updated migration.

**Review by solnsubuga — APPROVED** [review]

**Review by wasswa-derick — APPROVED** [review]
LGTM

File: wger/manager/migrations/0010_auto_20180809_1739.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    dependencies = [
+        ('manager', '0009_workoutlog_session_id'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='workoutlog',
+            old_name='session_id',
+            new_name='session',
```
Comment (RrNn) [inline wger/manager/migrations/0010_auto_20180809_1739.py:18]: @solnsubuga, right there

**Pull request merged**